packages/legend-graph/src/index.ts

```diff
@@ -233,7 +233,10 @@ export {
   V1_stereotypePtrSchema,
   V1_taggedValueSchema,
 } from './models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_DomainSerializationHelper';
-export { V1_serializeValueSpecification } from './models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ValueSpecificationSerializer';
+export {
+  V1_serializeValueSpecification,
```

Comment:
Lets try and not expose these. `V1` should never be used in anything but builders.